version/version.go

```diff
@@ -1,4 +1,4 @@
 package version
 
 // Version is the version of the build.
-const Version = "0.2.2-dev"
+const Version = "0.2.3-dev"
```

Comment:
Wait, I think this is correct, and what I said on IRC was wrong - we should leave this as 0.2.2-dev and change podman.spec to match. Sorry for the confusion

Reply:
ok reverted